### DISCUSSION TITLE
Docking station issues

### DISCUSSION THREAD

**Discussion by arturasvlt** [rxhanson/RectanglePro-Community#522] [General]
Hi. So I am normally always using the docking station at work. While I am anjoying the rectangle pro a lot, I believe I dont understand something: How can I set everything up so that once I connect the docking station (via usb c) with two external monitors - the apps would open up in certain monitors. Ideally, if they are not open - rectangle would open them up. Because saving current layout while using the docking didn't work.

**Comment by rxhanson** [comment]
At the moment, saving a layout will only save everything on the current display. You can save multiple displays into separate layouts and drag all of the window configurations from one to another (tedious, I know - I have a backlog item for allowing multiple displays to be saved into one layout). 

Beyond that, you'll need to check a couple boxes: trigger on display connected and launch closed/minimized apps. For example:

![image](https://github.com/rxhanson/RectanglePro-Community/assets/13651296/8c5ee858-c931-44c4-ae4e-53364241f474)

FYI, with some docking stations / displays, the timing of the display addition can be obscured from Rectangle Pro, and it can cause layouts to fail to be applied. Guessing on this timing can cause some undesirable behavior, so if you end up in this scenario, it's better to just trigger the layout yourself when you know your display is ready then to rely on the "trigger on display connected".
